xdis/cross_dis.py

```diff
@@ -263,7 +263,7 @@ def op_has_argument(opcode: int, opc) -> bool:
     """
     Return True if `opcode` instruction has an operand.
     """
-    return opcode >= opc.HAVE_ARGUMENT
+    return opcode in opc.hasarg if hasattr(opc, "hasarg") else opcode >= opc.HAVE_ARGUMENT
```

Comment:
Based on your current investigation/information, what was there before is what should be used.